rmgpy/molecule/draw.py

```diff
@@ -1309,7 +1308,9 @@ def _render_atom(self, symbol, atom, x0, y0, cr, heavy_first=True, draw_lone_pai
             bounding_rect = [x1, y1, x2, y2]
 
             # Set color for text
-            if atom.element.isotope != -1:
+            if not isinstance(atom, Atom):
+                cr.set_source_rgba(0.0, 0.5, 0.0, 1.0)
+            elif atom.element.isotope != -1:
                 cr.set_source_rgba(0.0, 0.5, 0.0, 1.0)
```

Comment:
Since these two conditions run the same code, we could do `if not isinstance(atom, Atom) or atom.element.isotope != -1:` (check my order of operations but I think this is fine).

Reply:
Done!